bundle/direct/dresources/cluster.go

```diff
@@ -102,12 +102,24 @@ func (r *ResourceCluster) DoDelete(ctx context.Context, id string) error {
 }
 
 func (r *ResourceCluster) ClassifyChange(change structdiff.Change, remoteState *compute.ClusterDetails, _ bool) (deployplan.ActionType, error) {
+	changedPath := change.Path.String()
+	if changedPath == "data_security_mode" {
+		if change.Old == compute.DataSecurityModeDataSecurityModeStandard && change.New == compute.DataSecurityModeUserIsolation {
+			return deployplan.ActionTypeSkip, nil
+		}
+		if change.Old == compute.DataSecurityModeDataSecurityModeDedicated && change.New == compute.DataSecurityModeSingleUser {
+			return deployplan.ActionTypeSkip, nil
+		}
+		if change.Old == compute.DataSecurityModeDataSecurityModeAuto && (change.New == compute.DataSecurityModeSingleUser || change.New == compute.DataSecurityModeUserIsolation) {
```

Comment:
I suppose it's better to skip it than to remap these in RemapState. Separate question (follow up PR) - should we also return a reason in ClassifyChanges? So we can show reason="alias" next to action="skip" in the plan.

Reply:
Tbh I don't have a preference on adding the reason

Reply:
Separate reason in the plan would be nice. Can be separate PR.